### PR TITLE
Implement configurable packet_lifetime

### LINKED ISSUE #44: Make IBC Packet lifetime configurable
Currently hardcoded in `mesh-provider` and `mesh-consumer`:
- https://github.com/CosmWasm/mesh-security/blob/main/contracts/mesh-consumer/src/ibc.rs#L18
- https://github.com/CosmWasm/mesh-security/blob/main/contracts/mesh-provider/src/ibc.rs#L20

### PR DESCRIPTION
**Pull request by Art3miX**
We set the packet_lifetime on init of mesh-provider and mesh-consumer with a default.

Also exposed a msg to update it if needed in the future, left a TODO to set permission checks.

Closes #44

### CODE REVIEW

File: contracts/mesh-consumer/src/contract.rs [hunk 1]
```diff
@@ -70,12 +81,22 @@ pub fn execute_receive_rewards(
...
     };
 
     Ok(Response::default().add_message(msg))
 }
 
+pub fn execute_update_packet_lifetime(
+    deps: DepsMut,
+    time: u64,
+) -> Result<Response, ContractError> {
+    // TODO: do permissions check
```
Comment (JakeHartnell) [inline contracts/mesh-consumer/src/contract.rs:94]: If this is a `TODO` would rather merge without the ability to update the packet time, just have a sensible default. This is a vulnerability as is.

Comment (JakeHartnell) [reply contracts/mesh-consumer/src/contract.rs:94]: Please remove for now and then this is LGTM from me.

Comment (Art3miX) [reply contracts/mesh-consumer/src/contract.rs]: I placed it in comments, so we can just uncomment in the future if needed or delete.

File: contracts/mesh-provider/src/contract.rs [hunk 2]
```diff
@@ -319,6 +326,16 @@ pub fn execute_claim_rewards(
...
+// pub fn execute_update_packet_lifetime(
+//     deps: DepsMut,
+//     time: u64,
+// ) -> Result<Response, ContractError> {
+//     // TODO: do permissions check
+//     PACKET_LIFETIME.save(deps.storage, &time)?;
+//     Ok(Response::new()
+//     .add_attribute("method", "update_packet_lifetime"))
+// }
+
```
Comment (JakeHartnell) [inline contracts/mesh-provider/src/contract.rs:338]:
```suggestion
```

File: contracts/mesh-consumer/src/contract.rs [hunk 3]
```diff
@@ -70,12 +81,22 @@ pub fn execute_receive_rewards(
...
+// pub fn execute_update_packet_lifetime(
+//     deps: DepsMut,
+//     time: u64,
+// ) -> Result<Response, ContractError> {
+//     // TODO: do permissions check
+//     PACKET_LIFETIME.save(deps.storage, &time)?;
+//     Ok(Response::new()
+//     .add_attribute("method", "update_packet_lifetime"))
+// }
+
```
Comment (JakeHartnell) [inline contracts/mesh-consumer/src/contract.rs:99]:
```suggestion
```

File: contracts/mesh-provider/src/msg.rs [hunk 4]
```diff
@@ -65,6 +67,9 @@ pub enum ExecuteMsg {
     ClaimRewards {
         validator: String,
     },
+    // UpdatePacketLifetime {
+    //     time: u64
+    // }
```
Comment (JakeHartnell) [inline contracts/mesh-provider/src/msg.rs:72]:
```suggestion
```

File: packages/mesh-apis/src/consumer_execute.rs [hunk 5]
```diff
@@ -3,4 +3,5 @@ use cosmwasm_schema::cw_serde;
 #[cw_serde]
 pub enum ConsumerExecuteMsg {
     MeshConsumerRecieveRewardsMsg { validator: String },
+    // UpdatePacketLifetime { time: u64 },
```
Comment (JakeHartnell) [inline packages/mesh-apis/src/consumer_execute.rs:6]:
```suggestion
```

File: tests/src/bindings/MeshConsumer.client.ts [hunk 6]
```diff
@@ -27,3 +28,88 @@ export class MeshConsumerQueryClient implements MeshConsumerReadOnlyInterface {
...
+      {
+        update_packet_lifetime: {
+          time,
+        },
+      },
+      fee,
+      memo,
+      funds
+    );
+  };
```
Comment (JakeHartnell) [inline tests/src/bindings/MeshConsumer.client.ts:114]: Re-run? This should not be here.

**Review by JakeHartnell — APPROVED** [review]
LGTM

**Pull request merged**